packages/api/src/external/carequality/api.ts

```diff
@@ -9,14 +13,23 @@ const cqApiMode = Config.isProdEnv()
   : CQAPIMode.dev;
 
 /**
- * Creates a new instance of the Carequality API client.
- * @param apiKey Optional, API key to use for authentication. If not used, the API key will be retrieved from the environment variables.
+ * Creates a new instance of the Carequality Management API client.
  * @returns Carequality API.
  */
-export function makeCarequalityAPI(apiKey?: string): Carequality | undefined {
+export function makeCarequalityManagementAPI(): CarequalityManagementAPI | undefined {
   if (Config.isSandbox()) return;
-  const cqApiKey = apiKey ?? Config.getCQApiKey();
-  return new Carequality(cqApiKey, cqApiMode);
+  const cqApiKey = Config.getCQApiKey();
+  const cqOrgCert = Config.getCQOrgCertificate();
+  const cqPrivateKey = Config.getCQOrgPrivateKey();
+  const cqPassphrase = Config.getCQKeyPassphrase();
+
+  return new CarequalityManagementAPIImpl({
+    apiKey: cqApiKey,
+    apiMode: cqApiMode,
+    orgCert: cqOrgCert,
+    rsaPrivateKey: cqPrivateKey,
+    passphrase: cqPassphrase,
```

Comment:
nit: why not `password` everywhere instead of `passphrase`? we use `password` in our codebase

also, `rsaPrivateKeyPassword` will be more explicit - I don't know what `password` is for without context